website/docs/cdktf/concepts/fundamentals/providers.html.md

```diff
@@ -1,21 +1,27 @@
 ---
 layout: "docs"
-page_title: "Providers"
+page_title: "Providers and Resources"
 sidebar_current: "cdktf"
 description: "Providers allow Terraform to communicate with external APIs. Learn to define providers in a CDK for Terraform application."
 ---
 
-# Providers
+# Providers and Resources
 
 A [provider](https://www.terraform.io/docs/language/providers/index.html) is any external API that has a plugin for Terraform. Provider plugins like the [AWS provider](https://registry.terraform.io/providers/hashicorp/aws/latest)
 or the [cloud-init provider](https://registry.terraform.io/providers/hashicorp/cloudinit/latest/docs) act as a translation layer that allows Terraform to communicate with many different cloud providers, databases, and services.
 
 ![diagram: How Terraform uses plugins](images/terraform-plugin-overview.png)
 
-CDK for Terraform allows you to import both local providers and providers from the [Terraform Registry](https://registry.terraform.io/). The import process extracts the provider's schema and converts it into classes that you can use in your CDKTF application. This allows you to define resources for that provider in your preferred programming language.
+Terraform uses providers to provision [resources](https://www.terraform.io/docs/language/resources/index.html), which describe one or more infrastructure objects like virtual networks and compute instances. Each provider on the [Terraform Registry](https://registry.terraform.io/) has documentation detailing available resources and their configuration options.
 
+In your CDK for Terraform Application, you will use your preferred programming language to define the resources you want Terraform to manage on one or more providers. This page provides details about how to use providers and resources in your application and how to use [escape hatches](#escape-hatch) to change resource behavior when necessary.
 
-## Pre-built Providers
+
+## Providers
+
+You can use both local providers and providers from the Terraform Registry in your application. CDKTF generates the required code bindings from the providers you define in [`cdktf.json`](/docs/cdktf/create-and-deploy/configuration.html). This allows you to define resources for that provider in your preferred programming language.
+
+### Pre-built Providers
```

Comment:
```suggestion
### Install Pre-built Providers
```

Adding a verb to be consistent w/other headings (Import Providers, Use a Local Provider)